sources/api/migration/migrator/src/main.rs

```diff
@@ -82,11 +124,65 @@ fn run() -> Result<()> {
             process::exit(0);
         });
 
-    let migrations = find_migrations(
-        &args.migration_directories,
-        &current_version,
-        &args.migrate_to_version,
-    )?;
+    // DEPRECATED CODE BEGIN ///////////////////////////////////////////////////////////////////////
+    // check if the `from_version` supports signed migrations. if not, run the 'old'
+    // unsigned migrations code and return.
+    if !are_migrations_signed(&current_version) {
```

Comment:
```suggestion
```
I'd rather remove the code entirely when we don't want to support unsigned migrations.  I've found it's usually not worth having the branches, because they can go wrong on their own, you can forget them, etc.